web/templates/copy_message_button.hbs

```diff
@@ -1,3 +1,3 @@
-<button class="btn pull-right copy_button_base copy_message tippy-zulip-tooltip" data-tippy-content="{{t 'Copy and close' }}" aria-label="{{t 'Copy and close' }}" >
+<div class="copy_button_base copy_message tippy-zulip-tooltip" data-tippy-content="{{t 'Copy and close' }}" aria-label="{{t 'Copy and close' }}">
```

Comment:
Now that this is no longer a button, we should probably add `role=button` (see [MDN](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role)).

Reply:
Thanks, updated.